docker-compose.yml

```diff
@@ -53,10 +55,11 @@ services:
     build:
       <<: *base-build
       target: wp49-php72
+      args:
+        - WP49_PHP72_PORT=${WP49_PHP72_PORT}
     <<: *base-wordpress
     ports:
-      # Make sure this port matches the Dockerfile port override for wp49-php72.
-      - 8031:8031
+      - ${WP49_PHP72_PORT}:${WP49_PHP72_PORT}
```

Comment:
@Mwindo, looks like you forgot to update the `.env.template` and we should add these env vars to the REQUIRED_ENVVARS list
https://github.com/orgnc/wordpress-plugin/blob/master/dev.py#L18

Reply:
Good catch, thanks!

Reply:
Will do that now.

Reply:
https://github.com/orgnc/wordpress-plugin/pull/165